### ISSUE TITLE
demo site no longer live?

### ISSUE THREAD

**Issue by shanecranor** [sdesabbata/vis-a-wik#1]
Seems like the demo site link is broken

**Comment by stefdesabbata** [comment]
Hi, thanks for raising this. Yes, the demo has been offline for a few months, as there is an issue with some of the libraries. The project is now eight years old, and it would require updating or rewriting to be functional. Unfortunately, I am not sure when I will have the time to work on this.

**Comment by shanecranor** [comment]
That is odd, it seems to be working locally for me?

**Comment by stefdesabbata** [comment]
Yes, that is odd. Still, happy to know that it is still working somehow, at least if run locally. It might be something to do with the github pages environment I guess. I might try to look into it when I have some time.
